hummingbot/strategy/liquidity_mining/liquidity_mining.py

```diff
@@ -267,6 +300,9 @@ def create_base_proposals(self):
         return proposals
 
     def total_port_value_in_token(self) -> Decimal:
+        """
+        Portfolio value in token
```

Comment:
Total portfolio value in self._token amount

Reply:
Resolved in commit https://github.com/CoinAlpha/hummingbot/pull/3440/commits/b35ab311e01d62f7b1b468e7d6fb855932b78fa4